spec/lib/wifi_user/use_cases/check_user_is_sponsee_spec.rb

```diff
@@ -0,0 +1,32 @@
+describe WifiUser::UseCase::CheckUserIsSponsee do
+  subject { described_class.new }
+
+  let!(:normal_user) { FactoryBot.create(:user_details, :recent, :active, :self_signed) }
+  let!(:sponsored_user) { FactoryBot.create(:user_details, :recent, :active, :sponsored) }
```

Comment:
this line doesn't require `let!` Maybe even better put these lines in the 'before` block